diff --git a/includes/menu.inc b/includes/menu.inc
index b979275110..10046d94ab 100644
--- a/includes/menu.inc
+++ b/includes/menu.inc
@@ -1510,8 +1510,8 @@ function menu_tree_check_access(&$tree, $node_links = array()) {
     else {
       $select->condition('n.status', 1);
     }
-    $select->condition('n.nid', $nids, 'IN');
     $select->addTag('node_access');
+    $select->condition('n.nid', $nids, 'IN');
     $node_objects = $select->execute()->fetchAll();
     foreach ($node_objects as $node_object) {
       foreach ($node_links[$node_object->nid] as $mlid => $link) {
