diff --git a/node_view_permissions.module b/node_view_permissions.module
index 1fd8519..d8243c0 100644
--- a/node_view_permissions.module
+++ b/node_view_permissions.module
@@ -59,7 +59,7 @@ function node_view_permissions_node_access($node, $op, $account) {
         } 
         return NODE_ACCESS_ALLOW;
       }
-      if (!count(module_implements('node_grants'))) {
+      if (!user_is_logged_in() || !count(module_implements('node_grants'))) {
         return NODE_ACCESS_DENY;
       }
     }
