diff --git a/paragraphs.field_widget.inc b/paragraphs.field_widget.inc
index c110062..78cd36b 100644
--- a/paragraphs.field_widget.inc
+++ b/paragraphs.field_widget.inc
@@ -482,7 +482,7 @@ function paragraphs_field_widget_form_build(&$form, &$form_state, $field, $insta
             '#delta' => $delta,
             '#name' => implode('_', $parents) . '_collapse_button',
             '#type' => 'submit',
-            '#value' => t('Collapse'),
+            '#value' => t('Submit'),
             '#validate' => array(),
             '#submit' => array('paragraphs_collapse_submit'),
             '#limit_validation_errors' => array(),
